.github/ISSUE_TEMPLATE/0-bug-issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug Issue
+about: Use this template for reporting a bug
+---
+
+## System information
+- CPU information(`cat /proc/cpuinfo | grep "model name" | uniq`):
+- GPU information(`lspci -nn | grep -E 'VGA|isplay`):
+- Display server if rendering to display(X or wayland):
+
+## Issue behavior
```

Comment:
Hi, Sherry, need to describe the priority and impact of bugs.

Reply:
Priority should be defined by internal scrum